ScoutSuite/providers/azure/resources/sql/database_blob_auditing_policies.py

```diff
@@ -2,22 +2,26 @@
 
 from azure.mgmt.sql import SqlManagementClient
 
-from ScoutSuite.providers.base.configs.resource_config import ResourceConfig
+from ScoutSuite.providers.base.configs.resources import Resources
```

Comment:
Shouldn't this be `Resource`, instead of `Resources`?

Reply:
I replied here: https://github.com/nccgroup/ScoutSuite/pull/201#pullrequestreview-206907757